README.md

```diff
@@ -1,75 +1,8 @@
 # Mesop: Build delightful web apps quickly in Python 🚀
 
-### Used at Google for rapid internal app development
+If you're interested in learning how to use Mesop, please check out our main docs at: google.github.io/mesop/
```

Comment:
[question] does google.github.io/mesop automatically get rendered as a link?

------------

I think this is an interesting change. Do you think people that don't know about Mesop will land on the repo first or the github docs? 

One slight concern I have with this change to the docs is that we make the assumption that people will know what Mesop is. So the README here which for repositories is a kind of home page, feels less friendly now. 

I think the benefit of this change is you don't have to manage both the Docs and also this README home page which may have similar information. And the Docs can be more detailed since info can be spread across multiple pages.

I also wondering if people will click on the link if they don't know what Mesop is.

Reply:
that's fair, I kept the readme content and just added the links to the top (I think it linkifies, but i formatted it as markdown links just to make it look nicer)